src/zsl/application/service_application.py

```diff
@@ -179,6 +182,10 @@ def _configure_injector(self, modules):
         self.logger.debug("Injector configuration with modules {0}.".format(modules))
         self._dependencies_initialized = True
 
+    def _configure_flask_app(self):
+        if 'MAX_CONTENT_LENGTH' in self.config:
```

Comment:
Does it affect request body size for all type of task calls (console, worker task, http request)? If it affects only tasks called using a http request, I would prefer much more specific name, e.g. `HTTP_REQUEST_MAX_CONTENT_LENGTH`